pkgs/development/ocaml-modules/reason-react/default.nix

```diff
@@ -0,0 +1,24 @@
+{
+  buildDunePackage,
+  melange,
+  reason,
+  reason-react-ppx,
+}:
+
+buildDunePackage {
+  pname = "reason-react";
+  inherit (reason-react-ppx) version;
```

Comment:
```suggestion
  inherit (reason-react-ppx) version src;

```

Reply:
These can just be one line :)

Reply:
Ok, I didn't know which coding style to prefer here.